include/rocksdb/comparator.h

```diff
@@ -78,6 +84,19 @@ class Comparator {
   // The major use case is to determine if DataBlockHashIndex is compatible
   // with the customized comparator.
   virtual bool CanKeysWithDifferentByteContentsBeEqual() const { return true; }
+
+  size_t TimestampSize() const { return timestamp_size; }
```

Comment:
It's not required but simple accessors can be named as `timestamp_size()`: https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
Done.